src/Serializer/DefaultSerializer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2021 Kai Sassnowski
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/ksassnowski/venture
+ */
+
+namespace Sassnowski\Venture\Serializer;
+
+use Laravel\SerializableClosure\Serializers\UnserializeException;
+
+final class DefaultSerializer implements WorkflowJobSerializer
```

Comment:
Is the default serializer needed since the `Base64WorkflowSerializer` operates the same except when dealing with Postgre connections?

Reply:
That mostly exists for testing purposes so we don’t have to fiddle around with connection objects everywhere we depend on a serializer. I started calling it `FakeSerializer` but realized it doesn’t actually fake anything 😅

Reply:
Ohh okay understandable! Sounds good 👍